composer/loggers/tensorboard_logger.py

```diff
@@ -55,6 +56,9 @@ def __init__(self, log_dir: Optional[str] = None, flush_interval: int = 100, ran
         self.flush_interval = flush_interval
         self.rank_zero_only = rank_zero_only
         self.writer: Optional[SummaryWriter] = None
+        self.flush_count = 0
+        self.event_file_base_file_path: Optional[str] = None
```

Comment:
Curious do we need to bind this variable to `self`? It seems to be read only in the same function where it is set.